package.json

```diff
@@ -47,5 +48,14 @@
     "proxyquire": "^2.1.3",
     "sinon": "^9.2.4"
   },
-  "funding": "https://opencollective.com/serialport/donate"
+  "funding": "https://opencollective.com/serialport/donate",
+  "changelog": {
+    "labels": {
+      "breaking": ":boom: BREAKING CHANGES :boom:",
+      "feature-request": "Features",
+      "bug": "Bug Fixes",
+      "docs": "Documentation",
+      "internal": "Chores"
```

Comment:
They are GitHub labels... the Lerna changelog seems to be based on PR labels in GitHub, rather than commit prefixes

So if we set the label as "internal" then they are grouped under "chore" in the changelog.

Since I can't seem to get it to work with historic records there is only one PR to play with for now... (you can change the label etc for that PR and see how it shows in the log).

Ps - If you think it is clearer then we could rename the "internal" label as "chore" for consistency